portable/GCC/NiosII/port_asm.S

```diff
@@ -74,24 +74,13 @@ save_sp_to_pxCurrentTCB:
 	movia	et, pxCurrentTCB	# Load the address of the pxCurrentTCB pointer
 	ldw		et, (et)			# Load the value of the pxCurrentTCB pointer
 	stw		sp, (et)			# Store the stack pointer into the top of the TCB
-	
-	.section .exceptions.irqtest, "xa"	
-hw_irq_test:

```

Comment:
why was this section deleted?

Reply:
I don't immediately understand the remaining section replacements, deletions, relocations, etc.
I'll review your info once more, and try to answer my own questions.

Reply:
This section was not removed. This is written [line 80](https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/41/files#diff-3c3df6855cab213c10b04eaeea93f554e05ec5ae1b1a8d8f3d4de56b7680c5c1R80)